src/chocolatey/infrastructure.app/services/ChocolateyConfigSettingsService.cs

```diff
@@ -55,14 +55,14 @@ public IEnumerable<ChocolateySource> source_list(ChocolateyConfiguration configu
                         source.Id,
                         source.Disabled ? " [Disabled]" : string.Empty,
                         source.Value,
-                        string.IsNullOrWhiteSpace(source.UserName) ? string.Empty : "(Authenticated)",
+                        string.IsNullOrWhiteSpace(source.UserName) && string.IsNullOrWhiteSpace(source.Certificate) ? string.Empty : "(Authenticated)",

```

Comment:
Yeah no worries. I'm not sure a username without a password can be put in, so it's likely just that it works by association.